.github/s2n_freebsd.sh

```diff
@@ -15,7 +15,12 @@
 set -eu
 export CTEST_PARALLEL_LEVEL=$(sysctl hw.ncpu | awk '{print $2}')
 
+cmake . -Brelease -GNinja -DCMAKE_BUILD_TYPE=Release
```

Comment:
Do we still need the debug build anymore?

Reply:
I don't really see any reason to remove it. The job is a little slower, but definitely not our slowest CI build.